bindings/nodejs/types/account.ts

```diff
@@ -50,6 +58,7 @@ export interface AccountSyncOptions {
     syncOnlyMostBasicOutputs?: boolean;
 }
 
+/** The account object */
```

Comment:
account of account metadata ?

Reply:
It's not account metadata, it's the account, don't know why it's called meta

Reply:
Should we rename it ?

Reply:
I would say yes, but let's ask Firefly team first, they came up with this name